app/models/profile.rb

```diff
@@ -99,31 +99,26 @@ def from_omniauth_hash(omniauth_user_hash)
     self.attributes.merge(update_hash){|key, old, new| old.blank? ? new : old}
   end
 
-  def image_url= url
-    return image_url if url == ''
-    if url.nil? || url.match(/^https?:\/\//)
-      super(url)
-    else
-      super(absolutify_local_url(url))
+  def _set_image_url (att, url)
```

Comment:
If you want to make a private method, just move it to the private methods, do not use silly names like starting with an underscore. Besides, Ruby naming conventions already use preceding underscores for unused arguments.